packages/component/tsup.config.ts

```diff
@@ -6,18 +6,24 @@ import { componentStyleContent as componentStyleContentPlaceholder } from './src
 import { decoratorStyleContent as decoratorStyleContentPlaceholder } from './src/decorator/private/createStyles';
 
 // TODO: [P1] Compute this automatically.
-const DEPENDENT_PATHS = ['bundle/src/boot/exports/full.ts'];
+const DEPENDENT_PATHS = ['bundle/src/boot/exports/index.ts'];
 
 const commonConfig = applyConfig(config => ({
   ...config,
   entry: {
     'botframework-webchat-component': './src/index.ts',
-    'botframework-webchat-component.internal': './src/internal.ts',
-    'botframework-webchat-component.decorator': './src/decorator/index.ts'
+    'botframework-webchat-component.component': './src/boot/component.ts',
+    'botframework-webchat-component.decorator': './src/boot/decorator.ts',
+    'botframework-webchat-component.hook': './src/boot/hook.ts',
+    'botframework-webchat-component.internal': './src/boot/internal.ts'
   },
   esbuildPlugins: [
     ...(config.esbuildPlugins ?? []),
-    injectCSSPlugin({ stylesPlaceholder: componentStyleContentPlaceholder }),
+    injectCSSPlugin({
+      getCSSText: (_source, cssFiles) =>
```

Comment:
Question: what does the default logic wrong so it fails to resolve the CSS file name given the files almost identical?

Reply:
It was looping all output files and reached the `chunk-*.mjs` which contains the placeholder tag.

Then, it try to find `chunk-*.css` to copy the content but it failed. There are no `chunk-*.css` but only `botframework-webchat-component.*.css`.

So, nothing is copied.